### PR TITLE
Fix Test Failures

### PR DESCRIPTION
**Pull request by shweta83**

### CODE REVIEW

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_ec2.py

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_ec2.py
env:
    ROBOTTELO_server__deploy_arguments__deploy_sat_version: '6.14.0'
    ROBOTTELO_server__deploy_arguments__deploy_snap_version: '14.0'

File: tests/foreman/ui/test_computeresource_ec2.py [hunk 1]
```diff
@@ -45,7 +45,6 @@ def module_ec2_settings():
 
 @pytest.mark.tier2
 @pytest.mark.skip_if_not_set('http_proxy')
-@pytest.mark.skip_if_open("BZ:2032530")
```
Comment (lhellebr) [inline tests/foreman/ui/test_computeresource_ec2.py:48]: How does merely removing the conditional skip fix the issue? The BZ got closed so it's ok to remove but there is a workaround needed.

**Review by lhellebr — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_computeresource_ec2.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/ui/test_computeresource_ec2.py:48]: My local setup was not working so I was trying to find the actual failure with the PRT. There is an issue with the subnet selection so fixing that issue in this PR.

**Comment by Gauravtalreja1** [comment]
@shweta83 can you update this PR?

**Pull request closed without merging**